src/Services/ScriptService.php

```diff
@@ -77,4 +77,31 @@ public function gtmBody()
         $gtm .= '<!-- End Google Tag Manager (noscript) -->';
         echo $gtm;
     }
+
+    private function getGtmCotainerId()
+    {
+        if ($this->getEnvGtmContainerId() !== false) {
```

Comment:
Cant you just do:
```suggestion
return $this->getEnvGtmContainerId() ?: $this->getDbGtmCotainerId();
```
The code yo have now are calling the same function twice event though the output is still false if they fail.
makes sense?